tsconfig.json

```diff
@@ -1,6 +1,12 @@
 {
   "compilerOptions": {
     "module": "commonjs",
+    "lib": [
+      "DOM",
+      "ES5",
+      "ScriptHost",
```

Comment:
Yeah, that's a known issue with highlight.js - I believe we've mentioned it somewhere - maybe wherever we moved to `@types`. Fixing that is sadly contingent on fixing the definition in DefinitelyTyped or enable TypeScript to understand `browser` resolution/do conditional definition paths (since there's no `Node` constructor in node).